src/main/java/teamproject/CategorySort.java

```diff
@@ -0,0 +1,34 @@
+package teamproject;
+import java.util.ArrayList;
+public class CategorySort implements TestData {
+	//This method is for testing purposes.
+	//Coded with Strings for the time being to focus on the algorithm. Will later switch to structures that include actual recipes.
+	public static void main(String[] args) {
+		String[] recipesToDisplay = sortCategory();
+		for(int i = 0; i < recipesToDisplay.length; i++) {
+			System.out.println(recipesToDisplay[i]);
+		}
+	}
+	//Gets the recipe array.
+	public static String[][] getRecipes() {
+		return test;
+	}
+	//Gets the category the user wishes to browse.
+	public static String getCategory() {
+		return testCat;
+	}
+	//Uses ArrayList to shorten the full array to just the desired results.
+	public static String[] sortCategory() {
```

Comment:
Good news that I didn't realize when we were planning out this week's work.

DataStore's actually going to do the filtering for you, it will let you pull out only the "Entity"s with a category matching your desired category, so you don't have to do this part. DataStore can also handle sorting by a certain field for you, so eventually it'll be easy to sort by things like number of "likes".

You can instead assume all of your testdata will already be the correct Category, and simply return them.